egs/wsj/s5/steps/dict/train_g2p.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Copyright 2014  Johns Hopkins University (Author: Yenda Trmal)
+# Apache 2.0
+
+# Begin configuration section.  
+iters=5
+stage=5
+encoding='utf-8'
+remove_tags=true
+only_words=true
+icu_transform="Any-Lower"
+cmd=run.pl
```

Comment:
I think it's OK to remove the whole icu transform mess for the kaldi-wide scripts and perhaps just rely on "tr" or make it user's job?
@danpovey any opinion? I used that to correctly normalize casing of the lexicon (bc for example in Turkish, i is not lowercased I) and was handling utf-8 encoding well. But not sure if/how this is importat for kaldi in general